stl/inc/xutility

```diff
@@ -2843,16 +2843,16 @@ namespace ranges {
         using _Not_quite_object::_Not_quite_object;
 
         // clang-format off
-        template <input_or_output_iterator _It, sentinel_for<_It> _Se>
+        template <class _It, sentinel_for<_It> _Se>

```

Comment:
You really do not like the iterator concept

Reply:
Ironically enough removing the type-constraints on the iterator parameters came from @Quuxplusone, unprompted by me.